content/CORE/Tasks/Rsync.md

```diff
@@ -5,83 +5,83 @@ weight: 30
 
 {{< toc >}}
 
-Data often needs to be copied to another system for backup or when migrating to a new system.
+You often need to copy data to another system for backup or when migrating to a new system.
 A fast and secure way of doing this is by using [rsync](https://rsync.samba.org/).
-These instructions assume that TrueNAS systems are being used for both sides of the rsync task: *host* and *remote*.
+These instructions assume that both sides of the rsync task, host and remote, use a TrueNAS systems.
 
 ## Basic Requirements
 
-Rysnc requires a [dataset]({{< relref "Datasets.md" >}}) with the needed data on the *host* or *remote* system.
-Rsync provides the ability to either *push* or *pull* data.
-When using rsync to *push*, data is copied from a *host* system to a *remote* system.
-When using rsync to *pull*, data is pulled from a *remote* system and put on the *host* system.
+Rysnc requires a [dataset]({{< relref "Datasets.md" >}}) with the needed data on the host or remote system.
+Rsync provides the ability to either push or pull data.
+When using the **Rsync Tasks** function to push, data is copied from a host system to a remote system.
+When using **Rsync Tasks** function to pull, data is pulled from a remote system and put on the host system.
```

Comment:
When using the **Rsync Tasks** function to pull, it pulls data from a remote system and puts it on the host system.